presto-hive/src/main/java/io/prestosql/plugin/hive/HiveSessionProperties.java

```diff
@@ -91,7 +91,7 @@
     private static final String QUERY_PARTITION_FILTER_REQUIRED = "query_partition_filter_required";
     private static final String PROJECTION_PUSHDOWN_ENABLED = "projection_pushdown_enabled";
     private static final String TIMESTAMP_PRECISION = "timestamp_precision";
-    private static final String PARQUET_OPTIMIZED_WRITER_ENABLED = "parquet_optimized_writer_enabled";
+    private static final String PARQUET_OPTIMIZED_WRITER_ENABLED = "experimental_parquet_optimized_writer_enabled";
```

Comment:
If we do not care about backward compatiblity regarding session property maybe we can drop `@LegacyConfig` from config property too?

Reply:
i would care about backward compatibility, but there are no provisions in session properties for maintaining it.
in config properties there are, so i was able to do better there

Reply:
Well. You can have two session properties. And compute logical `OR` of those .... smelly I guess

Reply:
https://github.com/prestosql/presto/issues/5536